src/RedditInterfaces.ts

```diff
@@ -1,20 +1,20 @@
-interface RedditData {
+export interface IRedditData {
     kind: string,
-    data: ParentData
+    data: IParentData
 }
 
-interface ParentData {
+interface IParentData {
     modhash: string,
     dist: number,
-    children: Children[]
+    children: IChildren[]
 }
 
-interface Children {
+interface IChildren {
     kind: string,
-    data: ChildData
+    data: IChildData
 }
 
-interface ChildData {
+interface IChildData {
```

Comment:
i don't understand what are these types for totally ambiguous.

Reply:
you can ignore it. It was just a practice.

Reply:
@HumnaAleem1 Can we remove all the redundant types defined here ?